english/talk_to_neighbour/woman/go-with-scully/talk-aside.md

```diff
@@ -1,2 +1,5 @@
 Scully says "Listen, I need you're help. The world is in grave danger.
-Will you help me?"
\ No newline at end of file
+Will you help me?"
+
+[Help Scully](go-with-scully/help-scully/im-in.md)
+[Ignore Scully and keep walking](go-with-scully/ignore-scully/talk-to-alien.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.